security/cores.py

```diff
@@ -269,3 +287,10 @@ def is_active_menu_item(self, request, active_group):
             self.menu_group,
             'celerytaskrunlog',
         }
+
+    @short_description(_('related objects'))
+    def display_related_objects(self, obj, request):
+        return render_model_objects_with_link(
+            request,
+            [related_object.object for related_object in obj.related_objects.all() if related_object.object]
+        )
```

Comment:
maybe move the method into a mixin or an ancestor? the same code is used three times already